i18n/english.yml

```diff
@@ -1371,6 +1380,24 @@ components:
       custom: Custom
       noAccess: No Access
     save: Save
+  Validation:
+    agencyRequired: Field must be populated for feeds with more than one agency.
+    dateServiceIdCombinationDuplicate: Date (%exceptionDate%) and Service ID (%serviceId%) combination cannot appear more than once for all exceptions
```

Comment:
End all of these error message sentences with a period.

Reply:
Good call, updated in https://github.com/ibi-group/datatools-ui/pull/965#:~:text=to%20validation%20warnings-,cb8061a,-Merge%20state